extensions/cpfpFeeEstimator/src/extension.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { ApiVersion, Bitcoin, Network, TatumSDK } from '@tatumio/tatum'
+import { CpfpFeeEstimator } from './extension'
+
+describe('CPFP Fee Estimator', () => {
+  let tatumSdk: Bitcoin
+
+  beforeAll(async () => {
+    tatumSdk = await TatumSDK.init<Bitcoin>({
+      network: Network.BITCOIN,
+      configureExtensions: [CpfpFeeEstimator],
+      version: ApiVersion.V3,
+      //verbose: true,
+    })
+  })
+
+  afterAll(async () => {
+    await tatumSdk.destroy()
+  })
+
+  describe('estimateCPFPFee', () => {
+    it('should return additional fee', async () => {
```

Comment:
Won't the test fail once the tx is included in block?

Reply:
It will but that's the usual thing we have across those kinds of tests. Once someone needs the test to check for regression or just to quickly debug then they can grab a fresh Tx from mempool

Reply:
Thanks for confirming, I wasn't sure

Reply:
Thanks!